packages/@uppy/react/src/DashboardModal.js

```diff
@@ -74,7 +74,7 @@ class DashboardModal extends React.Component {
   }
 }
 
-DashboardModal.propTypes = { // Only check this prop type in the browser.
+DashboardModal.propTypes = {
```

Comment:
I think we should keep this comment.
```suggestion
DashboardModal.propTypes = { // Only check this prop type in the browser.
```

Reply:
I only believe in comments that add value and don't just explain the code. In this case the ternary is already explanatory.